tomcat/files/tomcat-default-Debian.template

```diff
@@ -13,26 +13,29 @@ TOMCAT{{ tomcat.ver }}_GROUP={{ tomcat.group }}
 # The home directory of the Java development kit (JDK). You need at least
 # JDK version 7. If JAVA_HOME is not set, some common directories for
 # OpenJDK and the Oracle JDK are tried.
+{%- if tomcat.java_home|default -%}
```

Comment:
I'm not familiar with [default filter](http://jinja.pocoo.org/docs/2.10/templates/#default)

If `tomcat.java_home` is undefined then what happens?  What condition will if check against here?

Reply:
In your link, there is `default(value, default_value=u'', boolean=False)`

so, `|default` or `|default('')` would be the same, ie it tests the empty string (which ends up as False where converted to a bool).



Now, you can have:
```
tomcat:
  lookup:
    catalina_base: False
    catalina_home: null
    catalina_tmpdir: 0
    catalina_pid: ''
    java_home: []
```

They would each evaluate to something false, and generate a comment.

Reply:
Thanks for clear explanation that `default` implies `default('')` .. I understand the intention.  Cool.